.gitignore

```diff
@@ -1,3 +1,15 @@
+# OS
+.DS_Store
+Thumbs.db
+
+# IDEs
+.buildpath
+.project
+.settings/
+.build/
+.idea/
+nbproject/
+
```

Comment:
Please remove these changes, I don't think we should have tool-specific .gitignores